srcpkgs/libjpeg-turbo/template

```diff
@@ -10,7 +10,7 @@ maintainer="Orphaned <orphan@voidlinux.org>"
 license="IJG, BSD-3-Clause, Zlib"
 homepage="https://libjpeg-turbo.org/"
 distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
```

Comment:
```suggestion
homepage="https://libjpeg-turbo.org/"
changelog="https://raw.githubusercontent.com/libjpeg-turbo/libjpeg-turbo/main/ChangeLog.md"
distfiles="${SOURCEFORGE_SITE}/${pkgname}/${pkgname}-${version}.tar.gz"
```

Reply:
I applied the suggested changes and force-pushed it.